src/thrift_0_13_0/patch/0001-Disable-unneeded-packages.patch

```diff
@@ -0,0 +1,112 @@
+From cfefc0c946c999838926298a01d84c2af1905be7 Mon Sep 17 00:00:00 2001
+From: user <user@contoso.com>
+Date: Tue, 10 Aug 2021 15:01:25 +0000
+Subject: [PATCH] Disable-unneeded-packages
+
+---
+ debian/control | 54 +++++++++++++++++++++++++-------------------------
+ debian/rules   |  7 +++----
+ 2 files changed, 30 insertions(+), 31 deletions(-)
+
+diff --git a/debian/control b/debian/control
+index 5126e47..b0044da 100644
+--- a/debian/control
++++ b/debian/control
+@@ -13,8 +13,8 @@ Build-Depends: debhelper-compat (= 11), dh-python,
+  python3-all-dev,
+  python3-six,
+  golang-go, golang-github-golang-mock-dev,
+- pkg-php-tools (>= 1.14~), php-dev, phpunit, php-codesniffer,
+- perl (>= 5.22), libbit-vector-perl, libclass-accessor-perl,
++#  pkg-php-tools (>= 1.14~), php-dev, phpunit, php-codesniffer,
++#  perl (>= 5.22), libbit-vector-perl, libclass-accessor-perl,
```

Comment:
Look like you are commenting many code. Could you just delete instread of commenting. It will make patch more readable.

Reply:
Done